go/libkb/constants.go

```diff
@@ -98,18 +98,7 @@ var UserAgent = runtime.GOOS + ":" + "Keybase CLI (" + runtime.Version() + "):"
 
 // Returns a simplified UserAgent that's used as the kb_ua GET param.
 func ProofUserAgent() string {
-	var os string
-	if runtime.GOOS == "darwin" {
-		// Either ios or mac
-		if isIOS {
-			os = "ios"
-		} else {
-			os = "mac"
-		}
-	} else {
-		os = runtime.GOOS
-	}
-
+	os := runtime.GOOS
```

Comment:
Is this the same? Do we want to preserve the "mac" behavior?

Reply:
we can, don't think it's required though